packages/utils/src/miscellaneous.ts

```diff
@@ -0,0 +1,18 @@
+// import Logger from '@celo/mobile/src/utils/Logger';
+
+const TAG = 'utils/src/miscellaneous'
+
+export const reTryAsync = async (inFunction: any, tries: number, params: any) => {
+  let saveError
+  for (let i = 0; i < tries + 1; i++) {
+    try {
+      return await inFunction(...params)
```

Comment:
why `return await` here?

Reply:
I await here because I don't want to return the promise, I want the function to run otherwise it'd always do all the retries.

Reply:
Can you add a comment and maybe put the return on the next line to make this more clear?